### DISCUSSION TITLE
Automatically load favicons through DuckDuckGo

### DISCUSSION THREAD

**Discussion by MagicLike** [upptime/upptime#552] [Ideas]
It would be really nice, to load favicons automatically by default via DuckDuckGo, instead of searching and pasting each favicon individually in the `.upptimerc.yml`.
Here is for example the icon of DuckDuckGo:
https://icons.duckduckgo.com/ip3/www.duckduckgo.com.ico
![](https://icons.duckduckgo.com/ip3/www.duckduckgo.com.ico)

**Comment by AnandChowdhary** [comment]
You're right. We were using the GitHub favicon API which seems deprecated now, so we should totally switch to the Google or DuckDuckGo one!

**Reply by MagicLike** [reply]
But I wouldn't consider using the Google API due to privacy...

**Reply by MagicLike** [reply]
Where is the favicon API part located?

**Reply by AnandChowdhary** [reply]
https://sourcegraph.com/github.com/upptime/uptime-monitor/-/blob/src/summary.ts?L51

**Reply by MagicLike** [reply]
Openend a pull request here; https://github.com/upptime/uptime-monitor/pull/175

**Reply by AnandChowdhary** [reply]
Thanks!

**Reply by MagicLike** [reply]
No problem, I hope it works!

**Reply by MagicLike** [reply]
Is there a reason why the favicons are not displayed / looked up on my status page?

**Reply by MagicLike** [reply]
Any updates on this @AnandChowdhary?
Because there is still no change on the issue, that the favicons aren't shown on my status page.

**Reply by 4ht0h10** [reply]
I use this  and works fine for me: `icon: http://www.google.com/s2/favicons?domain=oracle.com`

**Reply by MagicLike** [reply]
> I use this and works fine for me: `icon: http://www.google.com/s2/favicons?domain=oracle.com`

It works fine now, at the time I commented, the release was not made...
